bedrock/firefox/templates/firefox/desktop/tips.html

```diff
@@ -12,15 +12,16 @@
 {% block body_id %}firefox-desktop-tips{% endblock %}
 {% block body_class %}firefox-desktop-tips{% endblock %}
 
-{% block site_css %}
+{% block page_css %}
   {% stylesheet 'firefox_desktop_tips' %}
 {% endblock %}
 
 {% block site_header %}
   <header id="masthead">
     <div class="container">
-      <a href="{{ url('mozorg.home') }}" id="tabzilla" data-infobar="{{ settings.TABZILLA_INFOBAR_OPTIONS }}">Mozilla</a>
-
+      <div id="tabzilla">
+        <a href="{{ url('mozorg.home') }}" data-infobar="{{ settings.TABZILLA_INFOBAR_OPTIONS }}">Mozilla</a>
```

Comment:
We no longer need the `data-infobar` attribute here, as static tabzilla doesn't support any additional infobar messaging.